extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -289,6 +312,7 @@ public void run()
       );
 
       started = true;
+      log.info(spec.toString());
```

Comment:
`log.info("%s", spec.toString())` is a little safer in case the spec string has some %'s in it. It may be better to combine this with the next line anyway though, since this log line is a bit anemic and context-free.